WoWsShipBuilder.Web/Services/DispersionPlotInterop.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Diagnostics.CodeAnalysis;
```

Comment:
Since this is basically one single method, put the JS directly to the component and move interop into the component, eliminating the need for this class entirely